imgui.go

```diff
@@ -686,3 +686,62 @@ func IsItemHovered() bool {
 func IsKeyPressed(key int) bool {
 	return C.iggIsKeyPressed(C.int(key)) != 0
 }
+
+// BeginColumns calls BeginColumnsV().
+func BeginColumns(count int, label string) {
+	BeginColumnsV(count, label, ColumnsFlagsNone)
+}
+
+// BeginColumnsV creates a column layout of the specified number of columns.
+func BeginColumnsV(count int, label string, flags int32) {
+	labelArg, labelFin := wrapString(label)
+	defer labelFin()
+	C.iggBeginColumns(C.int(count), labelArg, C.int(flags))
+}
+
+// NextColumn next column, defaults to current row or next row if the current row is finished.
+func NextColumn() {
+	C.iggNextColumn()
+}
+
+// GetColumnIndex get current column index.
+func GetColumnIndex() int {
+	return int(C.iggGetColumnIndex())
+}
+
+// GetColumnWidth get column width (in pixels). pass -1 to use current column.
+func GetColumnWidth() int {
+	return int(C.iggGetColumnWidth())
+}
+
+// SetColumnWidth calls SetColumnWidthV().
+func SetColumnWidth(index int, width float32) {
```

Comment:
If there are no optional parameters, there is no need to have a terse and a verbose variant. The `SetColumnWidthV` function can be removed. This function here can call C directly.